src/main/java/teammates/ui/request/FeedbackSessionRemindRequest.java

```diff
@@ -9,13 +9,20 @@ public class FeedbackSessionRemindRequest extends BasicRequest {
     private final String feedbackSessionName;
     private final String requestingInstructorId;
     private final String[] usersToRemind;
+    private final boolean isSendingCopyToInstructor;
 
     public FeedbackSessionRemindRequest(String courseId, String feedbackSessionName, String requestingInstructorId,
                                         String[] usersToRemind) {
+        this(courseId, feedbackSessionName, requestingInstructorId, usersToRemind, true);
```

Comment:
Do we need this version of `FeedbackSessionRemindRequest` constructor? It doesn't look complicated enough to warrant its own version.